StrictEmit/ILGeneratorExtensions.Variables.cs

```diff
@@ -359,5 +359,15 @@ public static void EmitUnalignedPrefix([NotNull] this ILGenerator il, Label inst
         [PublicAPI]
         public static void EmitUnalignedPrefix([NotNull] this ILGenerator il, byte offset)
             => il.Emit(OpCodes.Unaligned, offset);
+
+        /// <summary>
+        /// Declares a local of type T.
+        /// </summary>
+        /// <typeparam name="T">The type of the newly generated local.</typeparam>
+        /// <param name="il">The generator where the local is to be declared.</param>
+        /// <param name="pinned">true to pin the object in memory; otherwise false.</param>
```

Comment:
Extremly minor nitpick; the rest of the documentation writes "otherwise, false." instead of "otherwise false". Could you throw a comma in there?

Reply:
my bad - the il.DeclareLocal also includes a comma